src/woeusb

```diff
@@ -1,2060 +1,9 @@
-#!/usr/bin/env bash
-# A Linux program to create bootable Windows USB stick from a real Windows DVD or an image
-# Copyright © 2013 Colin GILLE / congelli501
-# Copyright © 2017 slacka et. al.
-#
-# This file is part of WoeUSB.
-#
-# WoeUSB is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# WoeUSB is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with WoeUSB  If not, see <http://www.gnu.org/licenses/>.
-#
-# Notes:
-# * Use --strip instead of --no-symlinks for compatibility of Ubuntu 14.04(EoL: April 2019)
-#
-# We use indirections and primitive variables, which is false positive of this rule
-# shellcheck disable=SC2034
+#!/usr/bin/python3
```

Comment:
The executable bit shouldn't be dropped.

Reply:
I probably have left it in core.py, as it was woeusb.py before, I'll move it. (You are talking about this comment on top of file, right?)